stumpy/snippets.py

```diff
@@ -10,6 +10,30 @@
 from .aampdist_snippets import aampdist_snippets
 
 
+def _get_mask_slices(idx, mask):
```

Comment:
Considering that this function is identical to the one in aampdist_snippets.py, I think this function should be moved into `core.py`